webrtc-audio-processing-sys/build.rs

```diff
@@ -16,8 +11,9 @@ mod webrtc {
     use failure::bail;
 
     const LIB_NAME: &str = "webrtc-audio-processing";
+    const LIB_MIN_VERSION: &str = "1.0.0";
```

Comment:
```suggestion
    const LIB_NAME: &str = "webrtc-audio-processing-1";
    const LIB_MIN_VERSION: &str = "1.0";
```

I had to make these changes for this to find my installed webrtc-audio-processing 1.0. But I made the distro package myself, and no other distro seems to package webrtc-audio-processing 1.0 yet, so I cannot confirm how the pkgconfig file name and version will look like.

Reply:
I checked the latest pkgconfig and they are actually naming it "webrtc-audio-processing-1". Thanks for the catch :)